packages/runtime/datastore/src/channelContext.ts

```diff
@@ -53,7 +54,16 @@ export function summarizeChannel(
     trackState: boolean = false,
 ): IChannelSummarizeResult {
     const summarizeResult = channel.summarize(fullTree, trackState);
-    // Add the channel attributes to the returned result.
+
+    // Add the channel attributes to the summary.
     addBlobToSummary(summarizeResult, attributesBlobKey, JSON.stringify(channel.attributes));
+
+    // Add GC data to the summary.
+    const gcDetails: IGCDetails = {
+        isRootNode: true,
+        gcNodes: summarizeResult.gcNodes,
+    };
+    addBlobToSummary(summarizeResult, gcBlobKey, JSON.stringify(gcDetails));
```

Comment:
Adding the gcNodes here means we are storing it twice since the data store will also store the nodes. This problem will be exacerbated once we have data stores within data stores. However, this is okay for now and is similar to the problem with storing outstandingOps.

Reply:
Not sure I fully follow. Are you saying that we need to duplicate it in a sense of storing per-channel info, and then having aggregated data store data at level up? If so, I think that's the intent, there is no other reasonable way around it that allows pay-per-play.
But the language in you comment suggest you have a plan to address it (or maybe it's different problem), if so, having an issue to track it would be great!